app/javascript/components/supplemental_application/SupplementalApplicationContainer.js

```diff
@@ -79,26 +80,29 @@ const ConfirmedPreferencesSection = ({ application, fileBaseUrl, onSave, confirm
   )
 }
 
-const ButtonPager = ({ disabled }) => (
-  <div className='button-pager'>
-    <div className='button-pager_row align-buttons-left primary inset-wide'>
-      <button className='button dropdown-button has-icon--right text-align-left small is-approved small has-status-width' href='#' aria-expanded='false' disabled={disabled}>
-        <span className='ui-icon ui-small' aria-hidden='true'>
-          <svg>
-            <use xlinkHref='#i-arrow-down' />
-          </svg>
-        </span>
-        Approved
-      </button>
-      <ul className='dropdown-menu' role='listbox' aria-hidden='true' aria-activedescendant='' tabIndex={0} style={{display: 'none'}}>
-        <li className='dropdown-menu_item' role='option' aria-selected='false'><a href='/some/valid/uri'>This is a link</a></li>
-        <li className='dropdown-menu_item' role='option' aria-selected='false'><a href='/some/valid/uri'>This is another</a></li>
-        <li className='dropdown-menu_item is-selected' role='option' aria-selected='true'><a href='/some/valid/uri'>Yet another</a></li>
-      </ul>
-      <button className='button primary small save-btn' type='submit' disabled={disabled}>Save</button>
-    </div>
-  </div>
-)
+const ButtonPager = withContext(({ loading, store }) => {
```

Comment:
@fddayan Does the name "ButtonPager" hold meaning for you? It doesn't mean anything to me, so I'm inclined to change it to something like "StatusDropdownWrapper", or something like that.

Reply:
I used ButtonPage since that is the name Jesse used for the class: `button-pager`.
I do not think `StatusDropdownWrapper ` makes sense since we have the status dropdown button AND the save button in there. The button page kind of represent the panel footer + panel actions.
I agree it's not super meaningful. Any other suggestion?

Reply:
Yes, good point, it's more than the status dropdown. I'll think on a meaningful name and propose something in a bit.

Reply:
After chatting with @fddayan we've decided to with `ActionButtons` for now. @fddayan had a good idea that in the future we could work with design to make a reusable buttons component that goes within the `ContentSection` component.